### ISSUE TITLE
Display HP in actor tooltip

### ISSUE THREAD

**Issue by michaeldgg2** [OpenE2140/OpenE2140#356]
Add text representation of actor's HP into actor's tooltip to mimick original game's context menu window

![Image](https://user-images.githubusercontent.com/119738087/250210070-703711a2-6309-47b8-95b2-e8cc3f87d229.png)

![Image](https://user-images.githubusercontent.com/119738087/250210199-b887cd7b-5765-4ee5-8d13-3083b5c5706d.png)

Also display an indicator, if vehicle under cursor is disabled.

**Comment by Dzierzan** [comment]
Is it really needed? Seems quite redundant as it is in the original game.

**Comment by michaeldgg2** [comment]
I don't think so. With it you can quickly assess how strong/weak a unit is. It is especially useful for people, who are not as familiar with units as we as devs or avid players are. Or maybe they are just starting out with OpenE2140.

**Comment by NarcolepticL1** [comment]
I'll start with this one
